code/src/components/Summary.js

```diff
@@ -0,0 +1,16 @@
+/* eslint-disable jsx-a11y/no-static-element-interactions */
+/* eslint-disable jsx-a11y/click-events-have-key-events */
+/* eslint-disable jsx-a11y/accessible-emoji */
+import React, { useState } from 'react'
+import { useSelector } from 'react-redux'
+
+export const Summary = () => {
+  const todos = useSelector((state) => state.todos.todosList)
+  const stillTodos = todos.filter((todo) => todo.done)
```

Comment:
if you're just interested in the length of these things, better to select exactly that.
`const todos = useSelector((state) => state.todos.todosList.length)`